spec/features/admin_can_create_products_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+feature "an admin can create products" do
+  before do
+    User.create(first_name: "Mike",
+                last_name: "Dorrance",
+                email: "mike@mike.com",
+                password: "12345678",
+                role: 1)
```

Comment:
we could be more clear by passing in "admin" to the role.  once we set up the enum, we get this functionality by default